apt/private/deb_translate_lock.bzl

```diff
@@ -23,43 +23,37 @@ _DEB_IMPORT_TMPL = '''\
     )
 '''
 
-_PACKAGE_TEMPLATE = """\
-alias(
-    name = "{target_name}",
-    actual = select({data_targets}),
-    visibility = ["//visibility:public"],
-)
-
-alias(
-    name = "control",
-    actual = select({control_targets}),
-    visibility = ["//visibility:public"],
-)
-"""
-
 _ROOT_BUILD_TMPL = """\
 "Generated by rules_distroless. DO NOT EDIT."
 
 load("@rules_distroless//apt:defs.bzl", "dpkg_status")
 
 exports_files(['packages.bzl'])
 
+# Map Debian architectures to platform CPUs.
+#
+# For more info on Debian architectures, see:
+#     * https://wiki.debian.org/SupportedArchitectures
+#     * https://wiki.debian.org/ArchitectureSpecificsMemo
+#     * https://www.debian.org/releases/stable/amd64/ch02s01.en.html#idm186
+#
+# For more info on Bazel's platforms CPUs see:
+#     * https://github.com/bazelbuild/platforms/blob/main/cpu/BUILD
 _ARCHITECTURE_MAP = {{
     "amd64": "x86_64",
-    "arm64": "aarch64",
-    "ppc64el": "ppc",
-    "mips64el": "mips64"
+    "armhf": "armv7e-mf",
```

Comment:
Can you send this as a separate PR?

Reply:
Ah, this is already in #120, it came in with the rebase :(